dali/operators/image/crop/crop.cc

```diff
@@ -32,27 +32,20 @@ DALI_SCHEMA(Crop)
         "image_type",
         R"code(The color space of input and output image)code",
         DALI_RGB, false)
+    .AddOptionalArg("out_of_bounds_policy",
+        R"code(Determines the policy when slicing out of bounds of the input.
+Supported values are:
+- \"error\" (default) : Attempting to slice outside of the bounds of the image will produce an error.
+- \"pad\": The input will be padded as needed with zeros or any other value specified with ``fill_values`` argument.
+- \"trim_to_shape\": The slice window will be resized so that it stays within the bounds of the input.
+a))code", "error")
+    .AddOptionalArg("fill_values",
+        R"code(Determines padding values, only relevant if ``out_of_bounds_policy`` is set to \"pad\".
+If a scalar is provided, it will be used for all the channels. If multiple values are given, there should be as many values as
+channels (extent of dimension 'C' in the layout) in the output slice.)code", std::vector<float>{0.f})
```

Comment:
Maybe this should be a part of `CropAttr`?

Reply:
I can't do that because then those will be exposed to ImageDecoderCrop as well and we don't want that for now. CropAttr just have crop attributes (crop_w, crop_h,...). What I can do is create a new XXXXAttr schema just for `out_of_bounds_policy` and `fill_values` that will be reused in Slice/Crop and any other operator that could implement out of bounds sampling

Reply:
Sounds good, and when the ImageDecoderCrop supports it we can move.

Reply:
I introduced OutOfBoundsAttr, just to have those two arguments in a single place